internal/controllers/device_config_controller.go

```diff
@@ -386,6 +392,11 @@ func (d *DeviceConfigController) GetConfigURLsFromVIN(c *fiber.Ctx) error {
 		ud = &pb.UserDevice{
 			DeviceDefinitionId: definitionResp.DeviceDefinitionId,
 		}
+		if len(definitionResp.DeviceStyleId) > 0 {
+			ud.DeviceStyleId = &definitionResp.DeviceStyleId
+		}
+		// todo: get powertrain type from definition response and include in ud.PowerTrainType
```

Comment:
not quite sure how to do this here, definitionResp doesn't directly have access to powertrain